inscriptions/ISic000092.xml

```diff
@@ -204,41 +209,72 @@
+                    <lb n="1" xml:id="AJUAK"/><persName type="attested" xml:id="AJUAU"><name xml:id="AJUAe"><w n="5"><expan xml:id="AJUAo"><abbr xml:id="AJUAy">C</abbr><ex xml:id="AJUAΙ">aio</ex></expan></w></name> <g ref="#ivy-leaf" xml:id="AJUAΤ" n="10">❦</g> <name xml:id="AJUAε"><w n="15">Maesio</w></name> <g ref="#ivy-leaf" xml:id="AJUAο" n="20">❦</g> <name xml:id="AJUBA"><w n="25">Aquillio</w></name>
+                    <lb n="2" xml:id="AJUBK"/><name xml:id="AJUBU"><w n="30">Fabio</w></name> <name xml:id="AJUBe"><w n="35"><hi rend="tall" xml:id="AJUBo">T</hi>i<hi rend="tall" xml:id="AJUBy">t</hi>iano</w></name></persName> <g ref="#interpunct" xml:id="AJUBΙ" n="40">·</g> <roleName type="supracivic" subtype="clarissimus" xml:id="AJUBΤ"><w xml:id="AJUBε" n="45"><expan xml:id="AJUBο"><abbr xml:id="AJUCA">c</abbr><ex xml:id="AJUCK">larissimo</ex></expan></w></roleName> <g ref="#interpunct" xml:id="AJUCU" n="50">·</g> <w xml:id="AJUCe" n="55"><expan xml:id="AJUCo"><abbr xml:id="AJUCy">v</abbr><ex xml:id="AJUCΙ">iro</ex></expan></w> <g ref="#interpunct" xml:id="AJUCΤ" n="60">·</g> <roleName type="supracivic" subtype="consul" xml:id="AJUCε"><w xml:id="AJUCο" n="65"><expan xml:id="AJUDA"><abbr xml:id="AJUDK">co</abbr><ex xml:id="AJUDU">n</ex><abbr xml:id="AJUDe"><hi rend="tall" xml:id="AJUDo">s</hi></abbr><ex xml:id="AJUDy">uli</ex></expan></w></roleName> <g ref="#ivy-leaf" xml:id="AJUDΙ" n="70">❦</g>
+                    <lb n="3" xml:id="AJUDΤ"/><w xml:id="AJUDε" n="75">optimo</w> <roleName type="civic" subtype="civis" xml:id="AJUDο"><w xml:id="AJUEA" n="80">civi</w></roleName> <w xml:id="AJUEK" n="85">ac</w> <roleName type="paracivic" subtype="patronus" xml:id="AJUEU"><w xml:id="AJUEe" n="90">patrono</w></roleName> <w xml:id="AJUEo" n="95">bene</w> <w xml:id="AJUEy" n="100">me
+                    <lb n="4" break="no" xml:id="AJUEΙ"/>renti</w> <g ref="#ivy-leaf" xml:id="AJUEΤ" n="105">❦</g> <space unit="character" quantity="3" xml:id="AJUEε"/><w xml:id="AJUEο" n="110">ordo</w> <w xml:id="AJUFA" n="115">et</w> <w xml:id="AJUFK" n="120">popu<hi rend="tall" xml:id="AJUFU">l</hi>us</w> <w xml:id="AJUFe" n="125">splen
+                    <lb n="5" break="no" xml:id="AJUFo"/><supplied reason="lost" xml:id="AJUFy">d</supplied>idis <hi rend="tall" xml:id="AJUFΙ">s</hi>imae</w> <placeName type="ancient" xml:id="AJUFΤ"><w xml:id="AJUFε" n="130"><expan xml:id="AJUFο"><abbr xml:id="AJUGA">col</abbr><ex xml:id="AJUGK">oniae</ex></expan></w> <w xml:id="AJUGU" n="135"><expan xml:id="AJUGe"><abbr xml:id="AJUGo">Aug</abbr><ex xml:id="AJUGy">ustae</ex></expan></w> <w xml:id="AJUGΙ" n="140">Himereorum</w>
```

Comment:
again, very picky, but there's an extra space in splendidissimae, before the hi element.